packages/lib/services/synchronizer/utils/handleConflictAction.ts

```diff
@@ -33,6 +33,7 @@ export default async (action: SyncAction, ItemClass: typeof BaseItem, remoteExis
 				changeSource: ItemChange.SOURCE_SYNC,
 				sourceDescription: 'sync: handleConflictAction: non-note conflict',
 				trackDeleted: false,
+				deleteChildren: false,
```

Comment:
I think that may be unexpected next time someone looks at this code. Could you please add a short comment above it?